### PR TITLE
Bbox for layers

### PR DESCRIPTION
**Pull request by pthorin**
Add a bbox for all allowed srs in the root layer.

### CODE REVIEW

**Comment by pthorin** [comment]
Closing due to staleness.

**Comment by manelclos** [comment]
@pthorin I'll try to have a look

File: ogcserver/wms111.py [hunk 1]
```diff
@@ -166,6 +166,19 @@ def GetCapabilities(self, params):
                 rootlayercrs.text = epsgcode.upper()
                 rootlayerelem.append(rootlayercrs)
 
+            for epsgcode in self.allowedepsgcodes:
```
Comment (manelclos) [inline ogcserver/wms111.py]: @pthorin just curious about why this is not in the previous `for` loop.

Comment (pthorin) [reply ogcserver/wms111.py]: Fair question. I don't see any reason why they can't be made into one single loop.

**Review by manelclos — APPROVED** [review]

**Pull request merged**